dashboard/views.py

```diff
@@ -467,19 +467,20 @@ def grade_distribution(request, course_id=0):
 
     df.sort_values(by=['current_grade'], inplace=True)
     df.reset_index(drop=True, inplace=True)
+    if df[df['current_grade'] > 100.0].shape[0] > 0:
+        summary['graph_upper_limit'] = int((5 * round(float(df['current_grade'].max()) / 5) + 5))
+    else:
+        df['current_grade'] = df['current_grade'].apply(lambda x: 99.99 if x == 100.00 else x)
+        summary['graph_upper_limit'] = 100
```

Comment:
Found a minor bug due to the +or -2 logic got affected. But it make sense to have this block of code before the binning logic if I re-evaluate again. Course: 501132

Reply:
Hm, okay. Should we document this bug as a separate issue? It can be resolved here.

I looked at the course you mentioned, I the trailing scores are being binned, but there is no indication that the last binning is everything below; should there be? We can discuss more on Monday.

Reply:
We talked about this in a meeting, feel free to resolve you in agreement.